src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -49,6 +49,16 @@ public without sharing class UTIL_CustomSettingsFacade {
     static List<Relationship_Auto_Create__c> relationshipAutocreate;
     static List<Relationship_Lookup__c> relationshipLookup;
 
+    // Constructor
+    public UTIL_CustomSettingsFacade() {
```

Comment:
It looks like some work was inadvertently committed here. Good catch, I missed that somehow. I need to clean this up.